src/main/java/shop/pet/track/dao/OwnerDAOImpl.java

```diff
@@ -0,0 +1,82 @@
+package shop.pet.track.dao;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.TypedQuery;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import shop.pet.track.entity.Owner;
+import shop.pet.track.entity.Pet;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Repository
+public class OwnerDAOImpl implements OwnerDAO {
+
+    EntityManager entityManager;
+
+    @Autowired
+    public OwnerDAOImpl(EntityManager entityManager) {
+        this.entityManager = entityManager;
+    }
+
+    @Override
+    public Owner findOwner(int id) {
+
```

Comment:
Again there are empty lines here and there in the entire code. Make them all consistent